save-frontend-common/src/main/kotlin/com/saveourtool/save/frontend/common/utils/LoginUtils.kt

```diff
@@ -110,5 +110,6 @@ fun mapKnownUploadedIcons(registrationId: String) =
             "gitee" -> "/img/gitee.svg"
             "github" -> "/img/github.svg"
             "google" -> "/img/google.svg"
+            "codehub" -> "/img/codehub.svg"
```

Comment:
I guess then we can delete `"codehub" -> faCopyright` at 96 line. Also as `faCopyright` if it's not used anymore

Reply:
That was my attempt to become a part time frontend developer
to remove it i think i need help @Cheshiriks. Just deleting them cause many problems

Reply:
The icon itself is still used in the Organization view. Therefore, here it is enough to delete only line 96

Reply:
done